### PR TITLE
Handle p and q values invalidation in the spreadsheet

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/components/network/equipment-table.js [hunk 1]
```diff
@@ -151,21 +157,34 @@ export const EquipmentTable = ({
...
                 <TableCell
                     component="div"
                     variant="body"
                     style={{ width: cellData.width }}
-                    className={classes.tableCell}
+                    className={
+                        classes.tableCell +
+                        ' ' +
+                        (canBeInvalidated &&
+                        loadFlowStatus !== RunningStatus.SUCCEED
```
Comment (laurgarn) [inline src/components/network/equipment-table.js:177]: Double negation, could be simplified, no ?

**Review by laurgarn — APPROVED** [review]

File: src/components/network/equipment-table.js [hunk 2]
```diff
@@ -151,21 +157,34 @@ export const EquipmentTable = ({
...
+            canBeInvalidated = undefined
+        ) => {
             const text = formatCell(cellData, numeric, fractionDigit, normed);
             return (
                 <TableCell
                     component="div"
                     variant="body"
                     style={{ width: cellData.width }}
-                    className={classes.tableCell}
+                    className={
```
Comment (sBouzols) [inline src/components/network/equipment-table.js:173]: Maybe use clsx here to compose multiple style classes here

Comment (FranckLecuyer) [reply src/components/network/equipment-table.js]: Done (and also in single line diagram component, where it was originally not done)

**Review by archendel — APPROVED** [review]
Tested when loadflow is not executed yet, failed or passed, in light and dark mode 
Worked as expected

**Pull request merged**